server/public/model/websocket_message.go

```diff
@@ -48,6 +48,7 @@ const (
 	WebsocketEventResponse                            = "response"
 	WebsocketEventEmojiAdded                          = "emoji_added"
 	WebsocketEventChannelViewed                       = "channel_viewed"
```

Comment:
This websocket event is not used anymore but I thought we shouldn't remove from the model, right? (for backwards compatibility and whatnot)

Reply:
Hmmm, not sure what our policy with regards to this is. @agnivade, any hints? If we're not using it, I guess we should deprecate it and eventually remove it.

Reply:
The new public module is still not 1.x, so we can do whatever we want. Feel free to remove.